### PR TITLE
#452

### PR DESCRIPTION
**Pull request by redaxmedia**
- style for byline
- added icons

### CODE REVIEW

File: includes/contents.php [hunk 1]
```diff
@@ -425,14 +425,19 @@ function byline($table, $id, $author, $date)
...
-		$output .= '<span class="rs-text-on"> ' . Redaxscript\Language::get('on') . ' </span>';
 	}
 
 	/* collect date and time output */
 
-	$output .= '<span class="rs-text-date">' . $date . '</span>';
-	$output .= '<span class="rs-text-at"> ' . Redaxscript\Language::get('at') . ' </span>';
-	$output .= '<span class="rs-text-time">' . $time . '</span>';
+	$output .= '<span class="rs-wrapper-text-byline">';
+		if ($table == 'articles')
```
Comment (redaxmedia) [inline includes/contents.php]: The indent for the if condition is wrong

File: includes/contents.php [hunk 2]
```diff
@@ -425,14 +425,19 @@ function byline($table, $id, $author, $date)
...
+		{
+			$output .= '<span class="rs-text-on"> ' . Redaxscript\Language::get('on') . ' </span>';
+		}
+		$output .= '<span class="rs-text-date">' . $date . '</span>';
+		$output .= '<span class="rs-text-at"> ' . Redaxscript\Language::get('at') . ' </span>';
+		$output .= '<span class="rs-text-time">' . $time . '</span>';
+	$output .= '</span>';
 
 	/* collect comment output */
```
Comment (redaxmedia) [inline includes/contents.php]: There should be a `<span class="rs-wrapper-comment-byline">` too!? Or we remove the divider span and keep the current classname

File: includes/contents.php [hunk 3]
```diff
@@ -425,14 +425,19 @@ function byline($table, $id, $author, $date)
...
 		$output .= '<span class="rs-text-by">' . Redaxscript\Language::get('posted_by') . ' ' . $author . '</span>';
-		$output .= '<span class="rs-text-on"> ' . Redaxscript\Language::get('on') . ' </span>';
 	}
 
 	/* collect date and time output */
 
-	$output .= '<span class="rs-text-date">' . $date . '</span>';
-	$output .= '<span class="rs-text-at"> ' . Redaxscript\Language::get('at') . ' </span>';
-	$output .= '<span class="rs-text-time">' . $time . '</span>';
+	$output .= '<span class="rs-wrapper-text-byline">';
```
Comment (redaxmedia) [inline includes/contents.php]: Rename this to `<span class="rs-wrapper-date-byline">`

File: templates/default/assets/styles/content.css [hunk 4]
```diff
@@ -56,10 +58,34 @@
...
+	margin: 1em 0 2em;
+}
+
+.rs-wrapper-text-byline
+{
+	display: block;
+	font-size: 0.875em;
+	opacity: 0.5;
+
+	&:before
```
Comment (redaxmedia) [inline templates/default/assets/styles/content.css]: Both icons via before should have a fixed width plus a text-align center ... the text after will start on the same position as a result

File: templates/default/assets/styles/content.css [hunk 5]
```diff
@@ -56,10 +58,34 @@
...
+	}
+}
+
+.rs-text-by:before
+{
+	@extend %rs-icon-person;
+	font-size: 1.5em;
+	margin-right: 0.25em;
+	vertical-align: middle;
 }
```
Comment (redaxmedia) [inline templates/default/assets/styles/content.css]: Styling for articles with a comment are missing

Comment (redaxmedia) [inline templates/default/assets/styles/content.css]: Add styline (icon) for comments too

File: includes/contents.php (hunk 1 above)
Comment (joergsteinhauer) [inline includes/contents.php]: what exactly? dont get it...

File: includes/contents.php (hunk 2 above)
Comment (joergsteinhauer) [inline includes/contents.php]: remove divider, no need anymore

**Comment by joergsteinhauer** [comment]
check this out, I added icons and all texts have the same size - no need for different sizes (less is more) https://gyazo.com/caedf1aef51ea9cf6b9fb8d1e2781235
also enlarged space between elements

**Comment by redaxmedia** [comment]
This issue never reached a version that is better than the original. Closed

**Pull request closed without merging**